service/controller/aws/v14/version_bundle.go

```diff
@@ -8,9 +8,9 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
-				Kind:        versionbundle.KindAdded,
+				Component:   "coredns",
+				Description: "Updated to 1.5.0. More info here: https://coredns.io/2019/04/06/coredns-1.5.0-release/",
```

Comment:
Shouldn't it be 1.5.0 in the PR title too?

On the changelogs is there a breaking change? We got the feedback last time it wasn't visible enough. Should we include here?

Not sure as we will cover in the release notes anyway.

Reply:
yes changed

Reply:
well It is the changelog of the app repo. Maybe it is good to link here to this changelog. And from there link upstream, WDYT?

Reply:
Yes good call. I think linking to our changelog is good.

Reply:
ok done